src/gt4py/next/program_processors/runners/dace_iterator/itir_to_sdfg.py

```diff
@@ -628,3 +754,33 @@ def _check_shift_offsets_are_literals(node: itir.StencilClosure):
             if not all(isinstance(arg, (itir.Literal, itir.OffsetLiteral)) for arg in shift.args):
                 return False
         return True
+
+    @staticmethod
+    def _replace_ssa_identifiers(closure: itir.StencilClosure):
+        """
+        Replace unicode symbols in function arguments with suffix identifiers based on the number of characters.
+
+        For example, 'z_gammaᐞ0ᐞ3' is renamed to 'z_gamma7_0'
+        Unicode symbols are not accepted in DaCe connectors, because C/C++ code does not have support.
+        """
+        _UNIQUE_NAME_SEPARATOR = "ᐞ"
+
+        def __replace_in_expression(fun: itir.FunCall, p_old: str, p_new: str):
+            for arg in fun.args:
+                if isinstance(arg, itir.FunCall):
+                    __replace_in_expression(arg, p_old, p_new)
+                elif isinstance(arg, itir.SymRef) and arg.id == p_old:
+                    arg.id = eve.SymbolRef(p_new)
+
+        if isinstance(closure.stencil, itir.Lambda):
+            for p in closure.stencil.params:
+                p_new = ""
```

Comment:
It think this will give a collision if both `z_gammaᐞ0ᐞ3` and `z_gamma7_0` are a parameter name.

Reply:
The for loop below should transform `z_gammaᐞ0ᐞ3` into `z_gamma7_010_3`

Reply:
True, but then `z_gammaᐞ0ᐞ3` collides with `z_gamma7_010_3`.

Reply:
I could add a check that `z_gamma7_010_3` does not already exist among `closure.stencil.params`, is that what you are trying to say?

Reply:
As a bandaid I'm fine with that, but we should find a general algorithm that is able to remove / escape special characters without collisions.

Reply:
I realized that this pass is not immediately needed. It is possible to enable `lift_mode=FORCE_TEMPORARIES` on gt4py unit tests without this pass. I can introduce it in a separate PR, later, if needed.